Drop the hard coded "gcc", also remove the -Werror since there are warnings with GCC 
author: fenglai liu (fenglai@accre.vanderbilt.edu)
Also remove CFLAGS: let EB handle it
updated by: Samuel Moors (Vrije Universiteit Brussel)
diff -ur SNAP-daf76badb477d22c08f2628117c00e057bf95ccf.orig/Makefile SNAP-daf76badb477d22c08f2628117c00e057bf95ccf/Makefile
--- SNAP-daf76badb477d22c08f2628117c00e057bf95ccf.orig/Makefile	2019-06-03 18:19:32.000000000 +0200
+++ SNAP-daf76badb477d22c08f2628117c00e057bf95ccf/Makefile	2021-02-20 14:26:34.782467000 +0100
@@ -62,7 +62,7 @@
 	cd Zoe; make clean
 
 depend: $(OBJECTS:.o=.c)
-	gcc $(INC) -MM $^ > $@
+	$(CC) $(INC) -MM $^ > $@
 
 tar:
 	rm -rf /tmp/$(APP)
@@ -79,9 +79,9 @@
 
 gcc:
 	cd Zoe; make;
-	make $(APP)  CC="gcc" CFLAGS="-O2 -Wall -Werror"
-	make $(APP2) CC="gcc" CFLAGS="-O2 -Wall -Werror"
-	make $(APP3) CC="gcc" CFLAGS="-O2 -Wall -Werror"
+	make $(APP)
+	make $(APP2)
+	make $(APP3)
 
 ###################
 # Inference Rules #
diff -ur SNAP-daf76badb477d22c08f2628117c00e057bf95ccf.orig/Zoe/Makefile SNAP-daf76badb477d22c08f2628117c00e057bf95ccf/Zoe/Makefile
--- SNAP-daf76badb477d22c08f2628117c00e057bf95ccf.orig/Zoe/Makefile	2019-06-03 18:19:32.000000000 +0200
+++ SNAP-daf76badb477d22c08f2628117c00e057bf95ccf/Zoe/Makefile	2021-02-20 14:27:30.710206000 +0100
@@ -47,7 +47,7 @@
 	rm -f *.o $(APP) $(APP2) $(APP3) $(APP4) $(APP5)
 
 depend: $(OBJECTS:.o=.c)
-	gcc -MM $^ > $@
+	$(CC) -MM $^ > $@
 
 tar:
 	rm -rf /tmp/$(APP)
@@ -63,7 +63,7 @@
 #################
 
 gcc:
-	make $(APP)  CC="gcc" CFLAGS="-O2 -Wall -Werror"
+	make $(APP)
 
 ###################
 # Inference Rules #
